### ISSUE TITLE
Bug: preview is broken for some builtins

### ISSUE THREAD

**Issue by Luxed** [nvim-lua/telescope.nvim#90]
For example: `builtin` and `buffers` are broken:
![image](https://user-images.githubusercontent.com/10234894/93413315-fe033180-f86c-11ea-9691-1309a0b36163.png)

This was tested on Windows.

**Comment by tjdevries** [comment]
Do you have bat installed? 
I may have busted some previewed today with a refractor, I'll take a look tomorrow.

Was it working before today for you?

**Comment by Luxed** [comment]
I have bat installed (it work perfectly well with `find_files` for example).

It was working under Linux before today, but it is the first time I test things on Windows, so I don't know if that's the issue.

**Comment by tjdevries** [comment]
I have a feeling I'm not escaping the file paths correctly. I will investigate tomorrow. Thanks for the helpful report.

**Comment by tjdevries** [comment]
Can you let me know if this is fixed?

**Comment by TimUntersberger** [comment]
This work for me now. Is it normal that the preview takes a long time to load?

**Comment by Luxed** [comment]
> This work for me now. Is it normal that the preview takes a long time to load?

On Linux it's blazing fast. So it might be an issue with Windows?
I've noticed that everything feels slower on Windows :confused:.
What do you qualify as "a long time"?
When I tested the `find_files` on Windows, I thought it was pretty fast.

**Comment by tjdevries** [comment]
@TimUntersberger What's the value of `&shell` for you? I wonder if there's something about spawning the new process that takes a long time on windows or something.

Perhaps I need to set up my windows computer for some testing... :) It seems I've got a few bugs left around there.

If it's working, let's close this issue and we can debug "slow preview times" in a new issue.

**Comment by TimUntersberger** [comment]
`shell` is set to `cmd.exe`.

Maybe we could have only one shell running in the background which gets reused instead of respawning every time?

**Comment by Luxed** [comment]
@TimUntersberger It doesn't look too bad to me.
What terminal emulator are you using? Or are you using `neovim-qt`?

> I guess I could also just write a custom previewer that uses a vim buffer instead of `bat`.

Judging by the comments in the code, I think this was already attempted? But it causes issues currently? I would also like this to be a feature since it looks weird with bat as it's not using the same colorscheme.

> If it's working, let's close this issue and we can debug "slow preview times" in a new issue.

That might be a good idea.

**Comment by Luxed** [comment]
@TimUntersberger I tried to reproduce the slowness on my Windows partition but I wasn't able to make it "as slow" as your gift.
I tried on all the terminal emulators I know of, and nothing felt that slow. I tried: alacritty, Windows Terminal (the new windows 10 store terminal), conemu and powershell. None of these was particularly slow.

I'm going to close this issue since it's solved and let you create a new one with more information regarding the slowness of the previewer.
